code/modules/shuttle/shuttle.dm

```diff
@@ -445,6 +460,12 @@
 		// This isn't an error, per se, but we can't let the shuttle code
 		// attempt to move us where we currently are, it will get weird.
 			return SHUTTLE_ALREADY_DOCKED
+	if(S.reservedId)
+		if(S.reservedId != id) // Checks so two shuttles don't get the same dock and conflict.
```

Comment:
S?.reservedId != id . You can do both test at once.

Reply:
I wanted to check if S.reserved was null or not and then check whether it's value doesn't match the mobile port id.
So i tried `S.reservedId? != id` but yeah thats not in dm.

Reply:
not S.reserverd? but S?.reserved